SPECS/liblzf/liblzf.spec

```diff
@@ -70,6 +69,9 @@ rm -f %{buildroot}%{_libdir}/liblzf.la
 %{_libdir}/pkgconfig/liblzf.pc
 
 %changelog
+* Tue Nov 01 2022 Riken Maharjan <rmaharjan@microsoft.com> - 3.6-24
```

Comment:
Let's also remove the comment block starting with "Leave lzcat  out since it conflicts with xz-lzma". It looks like the command it's referring to is commented out as well.